pkg/storage/unified/resource/search.go

```diff
@@ -508,6 +535,54 @@ func (s *searchSupport) monitorIndexEvents(ctx context.Context) {
 	}
 }
 
+func (s *searchSupport) startPeriodicRebuild(ctx context.Context) {
+	ticker := time.NewTicker(s.rebuildInterval)
+	defer ticker.Stop()
+
+	s.log.Info("starting periodic index rebuild", "interval", s.rebuildInterval)
+
+	for {
+		select {
+		case <-ctx.Done():
+			s.log.Info("stopping periodic index rebuild due to context cancellation")
+			return
+		case <-ticker.C:
+			s.log.Info("starting periodic index rebuild")
+			if err := s.rebuildAllIndexes(ctx); err != nil {
```

Comment:
🤔  `rebuildAllIndexes` is odd when IIUC we are only rebuilding dashboards periodically

Reply:
good point - renamed it